src/seam-os/client.ts

```diff
@@ -180,6 +182,7 @@ export class SeamOS {
   public readonly buildings = {
     create: this._postGrab("/buildings/create", "building"),
     get: this._getGrab("/buildings/get", "building"),
+    get_counts: this.get("/buildings/get_counts"),
```

Comment:
I couldn't use `_getGrab` here. Should I be using `.get` instead?

Reply:
You have to use _getGrab or the types will not work

Reply:
What error are you seeing?

Reply:
See here:
![Screen Shot 2023-01-15 at 7 35 53 PM](https://user-images.githubusercontent.com/1843707/212593302-4f720799-553f-4435-a311-52b383c88e16.png)

Reply:
the `/access_passes/get_counts` route on the backend is incorrect and should be fixed. It should expose `{ access_pass_counts }`

Reply:
all the counts endpoints seem to have this issue

Reply:
I'm investigating this

Reply:
kk, i'll leave this open until we address the backend issues. thx~